ResourceStatusSystem/Client/ResourceStatusClient.py

```diff
@@ -46,6 +46,12 @@ def _prepare(self, sendDict):
 
     return sendDict
 
+  def insert(self, tableName, record):
+    """
+    Insert a dictionary `record` as a row in table `tableName`
```

Comment:
Add docstring and `:params:` (see several examples in the code). Also add an example of `record`.